openassessment/xblock/openassessmentblock.py

```diff
@@ -720,6 +738,7 @@ def parse_xml(cls, node, runtime, keys, id_generator):
         block.allow_latex = config['allow_latex']
         block.leaderboard_show = config['leaderboard_show']
         block.group_access = config['group_access']
+        block.labels = config['labels']
```

Comment:
This might fail for the exported ORAs where the labels field was not added.

Reply:
Fixed in cbd3912d8ff982c42c927b70a70cc24e6ff47055

Also added checks for None and an empty string value.